File: app/src/main/java/trunk/doi/base/ui/fragment/mine/mvp/MinePresenter.java
package trunk.doi.base.ui.fragment.mine.mvp;

import android.app.Application;

import com.base.lib.di.scope.FragmentScope;
import com.base.lib.mvp.BasePresenter;
import com.base.lib.rx.RxBindManager;

import java.util.List;

import javax.inject.Inject;

import io.reactivex.Observer;
import io.reactivex.disposables.Disposable;
import trunk.doi.base.bean.GankItemData;

@FragmentScope
public class MinePresenter extends BasePresenter<MineModle, MineContract.View> {


    @Inject
    Application mApplication;

    @Inject
    public MinePresenter(MineModle model, MineContract.View rootView) {
        super(model, rootView);
    }

    @Override
    public void onDestroy() {
        super.onDestroy();

    }

}

File: module_zhihu/src/main/java/com/gas/zhihu/ui/fragment/mine/mvp/MinePresenter.kt
package com.gas.zhihu.ui.fragment.mine.mvp

import android.app.Application
import com.base.lib.di.scope.FragmentScope
import com.base.lib.mvp.BasePresenter
import javax.inject.Inject

@FragmentScope
class MinePresenter @Inject constructor(model: MineModle?, rootView: MineContract.View?) : BasePresenter<MineModle?, MineContract.View?>(model, rootView) {
    @JvmField
    @Inject
    var mApplication: Application? = null
    override fun onDestroy() {
        super.onDestroy()
    }
}